sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -261,6 +266,50 @@ def _set_partition_key(self, value):
             self.message.header = header
 
 
+class EventDataBatch(_BatchSendEventData):
```

Comment:
Is it better to have one class instead of two with inheritance?

Reply:
Good idea. I've done that.
Take a look at the new design.